courses/machine_learning/deepdive2/launching_into_ml/solutions/python.BQ_explore_data.ipynb

```diff
@@ -14,15 +14,15 @@
     "## Learning Objectives\n",
```

Comment:
<p>Good job on catching the "typos". </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/1118/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>